apps/catalog-admin/pages/api/code-lists/[...slug].ts

```diff
@@ -0,0 +1,46 @@
+import { createCodeList, deleteCodeList, getAllCodeLists, patchCodeList } from '@catalog-frontend/data-access';
+import { NextApiRequest, NextApiResponse } from 'next';
+import { getToken } from 'next-auth/jwt';
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+  const token = await getToken({ req });
+  const { slug } = req.query;
+  const [catalogId, codeListId] = slug;
+
+  if (req.method == 'GET') {
+    getAllCodeLists(catalogId as string, `${token?.access_token}`)
+      .then((response) => {
+        return res.status(200).send(response);
+      })
+      .catch(() => {
+        return res.status(500).send('');
+      });
+  } else if (req.method == 'POST') {
+    const { codeList } = JSON.parse(req.body);
```

Comment:
Hvis du for eks ser på concept-catalog catalog-admin api så ser du litt mer lesbar oppsett